xml_converter/attribute_test_script.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+ATTRIBUTE_TEST_CASES_FOLDER_PATHS=("./test_cases/can_fade_tests")
```

Comment:
Lets get this data dynamically to avoid the toil of having to edit this script over and over every time a new testcase is added.